project/dexter-core/src/test/com/samsung/sec/dexter/core/config/PeerReviewConfigFileTest.java

```diff
@@ -0,0 +1,66 @@
+package com.samsung.sec.dexter.core.config;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.samsung.sec.dexter.core.analyzer.ResultFileConstant;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.FileService;
+
+public class PeerReviewConfigFileTest {
+	PeerReviewConfigFile configFile;
+	FileService fileService;
+	
+	@Before
+	public void setUp() throws Exception {
+		fileService = mock(FileService.class);
+		configFile = new PeerReviewConfigFile(fileService);
+	}
+
+	@Test
+	public void checkDexterConfigMap_doNotCheckProjectFullPath() {
+		try {
+			// given
+			Map<String, Object> map = createTestConfigMap();
+			
+			// when
+			configFile.checkDexterConfigMap(map);
+		} catch (DexterRuntimeException e) {
+			// then
+			assert(false);
+		}
+	}
+
+	private Map<String, Object> createTestConfigMap() {
```

Comment:
I think we have to provide a validating method for the PeerReviewConfigFile class not only null/empty but also contents. For example the 'projectFullPath' should be PATH which is exist.